docs/platforms/javascript/guides/nuxt/manual-setup.mdx

```diff
@@ -50,66 +72,90 @@ yarn add @sentry/nuxt
   dsn: "___PUBLIC_DSN___",
+  // Replay may only be enabled for the client-side
+  integrations: [Sentry.replayIntegration()],
+
+  // Set tracesSampleRate to 1.0 to capture 100%
+  // of transactions for tracing.
+  // We recommend adjusting this value in production
+  // Learn more at
+  // https://docs.sentry.io/platforms/javascript/configuration/options/#traces-sample-rate
+  tracesSampleRate: 1.0,
+
+  // Capture Replay for 10% of all sessions,
+  // plus for 100% of sessions with an error
+  // Learn more at
+  // https://docs.sentry.io/platforms/javascript/session-replay/configuration/#general-integration-configuration
+  replaysSessionSampleRate: 0.1,
+  replaysOnErrorSampleRate: 1.0,
+});
+```
+
+We recommend you store your Sentry [Data Source Name](/concepts/key-terms/dsn-explainer/) (DSN) in an environment variable and configure it via the Nuxt runtime config like so:
 
-  // We recommend adjusting this value in production, or using tracesSampler
-  // for finer control
-  tracesSampleRate: 1.0
+```javascript {filename:nuxt.config.ts}
+export default defineNuxtConfig({
+  modules: ["@sentry/nuxt"],
+  runtimeConfig: {
+    public: {
+      sentry: {
+        dsn: process.env.SENTRY_DSN_PUBLIC, // Use a public environment variable for the DSN
+      },
+    },
+  },
 });
 ```
 
-### Server-side Setup
+This allows you to access the DSN using `useRuntimeConfig().public.sentry.dsn`.
+
+### Configure Server-side Sentry
 
-Add a `sentry.server.config.ts` file to the root of your project:
+Add a `sentry.server.config.ts` file to the root of your project and add the following initialization code to it:
 
 ```javascript {filename:sentry.server.config.ts}
-import * as Sentry from '@sentry/nuxt';
+import * as Sentry from "@sentry/nuxt";
 
 Sentry.init({
   dsn: "___PUBLIC_DSN___",
 
-  // We recommend adjusting this value in production, or using tracesSampler
-  // for finer control
-  tracesSampleRate: 1.0
+  // Set tracesSampleRate to 1.0 to capture 100%
+  // of transactions for tracing.
+  // We recommend adjusting this value in production
+  // Learn more at
+  // https://docs.sentry.io/platforms/javascript/configuration/options/#traces-sample-rate
+  tracesSampleRate: 1.0,
 });
 ```
 
-The Nuxt `useRuntimeConfig()` does not work in the Sentry server config due to technical reasons (the config file has to
-be loaded before Nuxt is loaded). To be able to use `process.env` you either have to add `--env-file=.env` to your node command
+We recommend you store your Sentry [Data Source Name](/concepts/key-terms/dsn-explainer/) (DSN) in an environment variable.
+
+<Expandable title="How to access environment variables in sentry.server.config.ts">
+Since Sentry on the server side needs to be loaded before `useRuntimeConfig()` is fully available, environment variables are not directly accessible via `process.env`. To make sure your environment variables are available, use one of these methods:
```

Comment:
The old docs are not very clear on this, but `useRuntimeConfig` and `process.env` are two different things. `useRuntimeConfig` is something Nuxt provides and as this is not available, people have to use `process.env`.

```suggestion
Since Sentry on the server side needs to be loaded before `useRuntimeConfig()` is fully available, environment variables are only accessible via `process.env`. To make sure your environment variables are available, use one of these methods:
```